### PR TITLE
fix: TR-858 settings menu separators issue due test running

### PR DESCRIPTION
**Pull request by lecosson**
**Related to:** https://oat-sa.atlassian.net/browse/TR-858

**Changes:**
- update req's for extension

**Requires:**
 - none

### CODE REVIEW

**Review by marpesia — APPROVED** [review]
Looks good!

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x]  Commits messages are meaningful
- [x]  Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Comment by meefox** [comment]
I have this error when running your brunch with LTR language. 
 
Screenshot 2021-06-09 at 18 16 09

File: views/js/controller/runner/runner.js
Comment (jsconan) [inline views/js/controller/runner/runner.js:180]: **nitpick:** The name is a bit confusing. It should rather highlight the relationship with RTL

**suggestion:** Instead of calling it `menuItemSelector`, better to name it `menuItemSelectorRTL` or something like that.

**Review by jsconan — CHANGES_REQUESTED** [review]
It works, but I wonder if this should not be done directly in the layout upfront, instead of in the client application. With the proposed solution, we can see a visual glitch as the page is always set in LTR while loading, and it stays as it for a few moments, then it finally applies RTL.

The header bar is built by the delivery container, as part of the main layout. A solution could be to work at [this level](https://github.com/oat-sa/extension-tao-delivery/blob/4f7f9773d95bdcf259acf328a1a32169bc5fab55/views/templates/DeliveryServer/blocks/header.tpl#L12).

Another possibility would be to do it through CSS only.

**Comment by jsconan** [comment]
> I have this error when running your brunch with LTR language.
> UPD: and RTL language too :)

@meefox This has been added by a previous PR, on[ tao-core-sdk](https://github.com/oat-sa/tao-core-sdk-fe/pull/109/files). If you have the error, this means you are not up to date. And this also means, the dependency didn't get updated in one of the previous PRs. @lecosson could you update accordingly the dependency to tao-core in the [composer file](https://github.com/oat-sa/extension-tao-testqti/blob/0e707cbc686e6576e5eb8abcc909f6a1849ab980/composer.json#L66)?

File: views/js/controller/runner/runner.js
Comment (meefox) [reply views/js/controller/runner/runner.js:180]: this is smth I want to try as well to use only CSS, if possible.

Comment (jsconan) [reply views/js/controller/runner/runner.js:180]: However, the CSS won't remove the visual glitch, as the RTL direction is set by the FE.
A better approach, would be definitely to set it from the BE directly on the template.

This may need some help from BE guys though. Eventually I can help adding the missing part.

File: composer.json
Comment (jsconan) [inline composer.json:66]: **nitpick:** While this requirement should not be harmful, it is not correct. The exact version that added the required feature is this one [v48.9.0](https://github.com/oat-sa/tao-core/releases/tag/v48.9.0). I think it should be set accordingly for later reference, so that we will able to have the right picture in several months if we need to figure out the changes made recently.

**Review by jsconan — COMMENTED** [review]
It could be ok, but I think we must be more cautious about the exact version we require. A more recent version that the one which included the required feature would be fine, but not accurate.

File: composer.json
Comment (meefox) [reply composer.json:66]: Hey @jsconan should we use v# or just a # in composer json or both are acceptable?

Comment (jsconan) [reply composer.json]: both are acceptable, but if possible better only use `#`

**Review by jsconan — APPROVED** [review]
I think you can merge this PR as soon as you get enough approvals. This one is about a technical change, so no need to wait for QA.

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**
